cmd/eudico/genesis.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/consensus-shipyard/go-ipc-types/types"
+
+	"github.com/filecoin-project/lotus/blockstore"
+	"github.com/filecoin-project/lotus/chain/vm"
+	"github.com/filecoin-project/lotus/genesis"
+	"github.com/filecoin-project/lotus/journal"
+	"github.com/filecoin-project/lotus/node/modules/testing"
+	"github.com/filecoin-project/lotus/storage/sealer/ffiwrapper"
+)
+
+var genesisCmd = &cli.Command{
+	Name:        "genesis",
+	Description: "manipulate lotus genesis template",
+	Subcommands: []*cli.Command{
+		genesisNewCmd,
+	},
+}
+
+var genesisNewCmd = &cli.Command{
+	Name:        "new",
+	Description: "create new genesis from the template and store it in a car file",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "subnet-id",
+			Value: types.RootStr,
+			Usage: "The ID of the subnet",
+		},
+		&cli.StringFlag{
+			Name:    "out",
+			Aliases: []string{"o"},
+			Value:   "genesis.car",
+			Usage:   "write output to `FILE`",
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		if cctx.Args().Len() != 1 {
```

Comment:
Instead of passing the genesis template an argument, how hard would it be to hardcode the template in the code, or at least let's use a default path within the repo and use a flag for the template so doing `./eudico genesis new --subnet-id` creates a new subnet ID template without the need of any additional arguments.

Reply:
Moved to `/genesis`.